superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -29,7 +29,7 @@ import { connect } from 'react-redux';
 
 // Session storage key for recent dashboard
 const SK_DASHBOARD_ID = 'save_chart_recent_dashboard';
-const SELECT_PLACEHOLDER = t('**Select** a dashboard OR **create** a new one');
+const SELECT_PLACEHOLDER = t('Select dashboard');
```

Comment:
Can we keep the original copy as I don't think we've added the functionality to create a dashboard in the dropdown yet. 

![image](https://user-images.githubusercontent.com/335541/104222987-e6415180-53f7-11eb-8a6b-72af901ff47c.png)

https://projects.invisionapp.com/share/V5VH03AHBER#/screens/399817652

cc @mihir174

For context, #10355 added this copy and there is a discussion on why this is still not optimal.

Reply:
@ktmud but the original text is referring to dashboard creation through save modal?

Reply:
Have we removed that functionality?

Reply:
It is possible to create a dashboard through this modal and I guess text should claim that creation is possible.